beamer/state_machine.py

```diff
@@ -523,13 +560,24 @@ def on_future_done(f: Future) -> None:
 
                 del context.l1_invalidations[claim.id]
             except Exception as ex:
-                log.error("L1 invalidation failed", ex=ex)
+                log.error(
+                    "L1 invalidation failed",
+                    ex=ex,
+                    source_chain_id=context.source_chain_id,
+                    target_chain_id=context.target_chain_id,
+                )
 
         future.add_done_callback(on_future_done)
         context.l1_invalidations[claim.id] = future
         claim.l1_invalidate()
 
-        log.info("Initiated L1 invalidation", request=request, claim=claim)
+        log.info(
+            "Initiated L1 invalidation",
+            request=request,
+            claim=claim,
+            source_chain_id=context.source_chain_id,
+            target_chain_id=context.target_chain_id,
+        )
```

Comment:
Please use the binding feature of `structlog`, which we already use elsewhere:
```python
self._log = structlog.get_logger(type(self).__name__).bind(chain_id=self._chain_id)
```
Bound values will be emitted with all the `log` calls, i.e. they are implicitly output. This would save us some lines and improve readability and maintenance.